aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -632,6 +657,14 @@ module aptos_framework::voting {
         timestamp::now_seconds() > proposal.expiration_secs
     }
 
+    inline fun get_proposal<ProposalType: store>(
```

Comment:
Let's make this a normal fun to reduce bytecode size. Inline functions save some gas (avoid 1 function call) but bloats bytecode size as it's expanded at every call site. With 5+ callsites, the disadvantage starts to make it not worth it :)

Reply:
@movekevin fixed